ini/features.ini

```diff
@@ -239,8 +240,8 @@ IS_SCARA                               = src_filter=+<src/module/scara.cpp>
 HAS_SERVOS                             = src_filter=+<src/module/servo.cpp> +<src/gcode/control/M280.cpp>
 MORGAN_SCARA                           = src_filter=+<src/gcode/scara>
 HAS_MICROSTEPS                         = src_filter=+<src/gcode/control/M350_M351.cpp>
-(ESP3D_)?WIFISUPPORT                   = AsyncTCP, ESP Async WebServer
-                                         ESP3DLib=https://github.com/luc-github/ESP3DLib/archive/master.zip
+(ESP3D_)?WIFISUPPORT                   = AsyncTCP, ESP Async WebServer=https://github.com/har-in-air/ESPAsyncWebServer.git
+                                         ESP3DLib=https://github.com/quiret/ESP3DLib
```

Comment:
You may want to submit your changes to @luc-github if that fork and branch is still being maintained, and if the changes are appropriate for all projects that use the `luc-github` fork.

Reply:
Please check https://github.com/luc-github/ESP3DLib/pull/51 I am in contact with @luc-github. May readjust things based on his recommendations!

Reply:
Just saw your other comments, thanks!